src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
+// debugger
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+function success() {
+  function positveResult(resolve) {
+    logo.addEventListener('click', () => {
+      resolve('Promise was resolved!');
+    });
+  }
+
+  return new Promise(positveResult);
+}
+
+function fail() {
+  const failed = (resolved, error) => {
```

Comment:
the first time you create an internal function via function declaration and the second time via function expression
be consistent in such things

Just FYI